adserver/utils.py

```diff
@@ -423,6 +423,13 @@ def generate_publisher_payout_data(
             )
         )
 
+    payouts_url = generate_absolute_url(
+        reverse("publisher_payouts", kwargs={"publisher_slug": publisher.slug})
+    )
+    settings_url = generate_absolute_url(
+        reverse("publisher_settings", kwargs={"publisher_slug": publisher.slug})
+    )
```

Comment:
This was used in email & in the payouts list page, so I moved it back inside the function instead of repeating it.